css/index.css

```diff
@@ -101,6 +144,8 @@ p {
     justify-content: space-evenly;
 }
 
+
+
```

Comment:
you may be able to target the boxes with just the .box in place of the entire chain... .middle-content .boxes .box